support/render/hal/link.go

```diff
@@ -10,11 +10,7 @@ type Link struct {
 }
 
 func (l *Link) PopulateTemplated() {
-	var err error
-	l.Templated, err = regexp.Match("{.*}", []byte(l.Href))
-	if err != nil {
-		panic(err)
-	}
+	l.Templated = strings.Contains(l.Href, "{")
```

Comment:
shouldn't be there an error checking?

Reply:
`Contains()` doesn't return an error at all. Only bool